common/djangoapps/student/urls.py

```diff
@@ -3,65 +3,58 @@
 """
 
 from django.conf import settings
-from django.conf.urls import patterns, url
+from django.conf.urls import url
+from django.contrib.auth.views import password_reset_complete
 
-from student.views import LogoutView
-
-urlpatterns = (
-    'student.views',
+from student.views import *
```

Comment:
not sure what our collective opinion is on importing star, especially when not everything is needed, e.g. `student.views.dashboard`.

Reply:
Nice catch, It Is Bad. Fix incoming.